include/shell/shell_bt_nus.rst

```diff
@@ -13,66 +13,74 @@ Setup
 
 Enable NUS shell transport in your application to be able to receive shell commands remotely.
 
-To send shell commands to an application that uses this module, you need specific software, comparable to a terminal (for example, PuTTY) for UART communication.
+To send shell commands to an application that uses this module, you need a serial console application, like PuTTY, for UART communication.
 
 .. _shell_bt_nus_host_tools:
 
 Sending shell commands
 **********************
 
-The |NCS| provides two alternatives for sending shell commands from a host (for example, a PC) to the application that uses this module.
-`bt_nus_shell.py`_ is a Python script that requires a terminal and a second development board.
-`BLE Console`_ is a stand-alone application for Linux.
+.. testing_general_start
+
+The |NCS| provides two alternatives for sending shell commands from a host, like a PC, to the application that uses this module:
+
+* `bt_nus_shell.py`_. It is a Python script that requires a serial console application, like PuTTY, and a second development kit.
+* `BLE Console`_. It is a stand-alone application for Linux.
+
+.. testing_general_end
 
 bt_nus_shell.py
 ===============
 
-The script file ``scripts/shell/bt_nus_shell.py`` contains a cross-platform example host application, written in Python 2.
+.. testing_bt_nus_shell_intro_start
+
+The script file :file:`scripts/shell/bt_nus_shell.py` contains a cross-platform example host application, written in Python 2.
 
-The script uses an additional Nordic Development Kit (for example, PCA10040) as a Bluetooth central device.
+The script uses an additional Nordic development kit, like the PCA10040, as a Bluetooth central device.
 It connects to the specified device and forwards all NUS traffic to the network port.
-You can then use a terminal application, for example PuTTY, to connect to that port (the default port is 8889) and use the shell.
+You can then use a serial console application, like PuTTY, to connect to that port and use the shell.
```

Comment:
`You can then use a console application`

Reply:
fixed